tests/devices/qubit/test_sampling.py

```diff
@@ -21,7 +21,7 @@
 from pennylane.devices.qubit import simulate
 from pennylane.devices.qubit import sample_state, measure_with_samples
 
-two_qubit_state = np.array([[0, 1j], [-1, 0]]) / np.sqrt(2)
+two_qubit_state = np.array([[0, 1j], [-1, 0]], dtype=np.complex128) / np.sqrt(2)
```

Comment:
Is it not working without np.complex128?

Reply:
complex64 is too inaccurate with finite diff.